conseil-common/src/test/scala/tech/cryptonomic/conseil/common/bitcoin/BitcoinDbViewsTest.scala

```diff
@@ -0,0 +1,67 @@
+package tech.cryptonomic.conseil.common.bitcoin
+
+import scala.concurrent.ExecutionContext
+
+import cats.effect._
+import org.scalatest.{Matchers, WordSpec}
+import slick.jdbc.PostgresProfile.api._
+
+import tech.cryptonomic.conseil.common.testkit.InMemoryDatabase
+
+class BitcoinDbViewsTest
+    extends WordSpec
+    with Matchers
+    with InMemoryDatabase
+    with BitcoinInMemoryDatabaseSetup
+    with BitcoinFixtures
+    with BitcoinStubs {
+
+  implicit val contextShift: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
+
+  "Bitcoin db views" should {
+
+      "return balance for the given address" in new BitcoinPersistenceStubs(dbHandler) {
+        val unspentOutput = RpcFixtures.outputResult.copy(
```

Comment:
It would  be great if we could have a description of the test logic, starting from how the data is prepared and how the expectations are met.

What are the script keys pointing at?
What is the relationship between different test data? Would it make more apparent if the values set when copying different objects would point to the other data fields?

e.g.
```scala
val block1 = Rpc.blockResult.copy(height = 1, hash = "hash1")
val tx1 = Rpc.transactionResult.copy(txid = "tx1", blockhash = block1.hash, vout = List(unspent))
val input = Rpc.inputResult.copy(txid = Some(tx1.txid), vout = Some(0))
...
```

Reply:
I've added an additional description